vault/dendron.topic.links.commands.convert-link.md

```diff
@@ -0,0 +1,7 @@
+---
+id: kywqudil465oriavcbmavlv
+title: Convert Link
+desc: ''
+updated: 1658831117794
+created: 1658831117794
+---
```

Comment:
for commands, we should have a loom video demonstrating the command

this command has no docs